modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -43,6 +43,10 @@
 }
 
 body .ddm-user-view-content {
+	.ddm-field-container {
+		padding: 0 12px;
```

Comment:
I think you should use 8 base values (like 8, 16, 24, ...) on portal's style

Reply:
Hey @miltonmc , thank you for your feedback!

In this case I used 12px to keep consistency with the padding present in the form builder:

https://github.com/liferay/liferay-portal/blob/3a8ab55f0466afd7405223636a5cef15307a31dd/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/css/form-builder/_form-builder.scss#L31

Also, 12px was the previous value used on the field padding in previous versions of the portal. Do you think we should we change both to 8px/16px or keep as 12px? Or something else?